youtrack_api_get/issue_info.py

```diff
@@ -0,0 +1,49 @@
+import httpx
+import datetime
+
+state_id = "111-2"
+priority_id = "111-0"
+time_end_id = "117-0"
+
+
+def get_custom_field(youtrack_domain, issue_name, field_id, token):
```

Comment:
Return `FieldData` entity, not just string

Reply:
(for me)
class FieldInfo(BaseModel):
    name: str
    value: str

Reply:
done